src/components/Table.js

```diff
@@ -1,22 +1,40 @@
 import { useMemo, useState } from "react";
 import { useGetUsersQuery } from "../utils/api";
 import { DataGrid } from "@mui/x-data-grid";
-import GenericButton from "./GenericButton";
-import VisibilityIcon from "@mui/icons-material/Visibility";
-import RemoveCircleIcon from "@mui/icons-material/RemoveCircle";
+import { Button } from "@mui/material";
+import { useRef } from "react";
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
 
 // -------------------------------------
 //  DATA GRID MUI
 // -------------------------------------
 
 const USERS = [];
 
-const handleRowClick = (params) => {
-  console.log(`User: ${params.row.uuid}`);
-};
-
 export default function UsersTable() {
   const { data = USERS, isLoading, refetch } = useGetUsersQuery();
+  const userListHelper = useRef(USERS);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    userListHelper.current = data;
+    // console.log(data);
```

Comment:
Hey, doc!
Try not to have any commented code in your apps.
Use comments only for documentation purposes.

Reply:
Sure, noted